apps/studio/src/features/mail/templates/templates.ts

```diff
@@ -74,6 +76,20 @@ export const loginAlertTemplate = (
   }
 }
 
+export const schedulePageTemplate = (
+  data: SchedulePageTemplateData,
+): EmailTemplate => {
+  const { recipientEmail, scheduledAt } = data
+  return {
+    subject: `[Isomer Studio] You scheduled a page to be published`,
```

Comment:
nit: can we add the `resource.title` here in the subject? similar to `publishAlertContentPublisherTemplate` - better UX IMO